src/NuGet.Core/NuGet.ProjectModel/PackageSpec.cs

```diff
@@ -185,24 +186,55 @@ public bool Equals(PackageSpec other)
         }
 
         /// <summary>
-        /// Clone a PackageSpec and underlying JObject.
+        /// Clone a PackageSpec
         /// </summary>
         public PackageSpec Clone()
         {
-            var writer = new JsonObjectWriter();
-            PackageSpecWriter.Write(this, writer);
-            var json = writer.GetJObject();
-
-            var spec = JsonPackageSpecReader.GetPackageSpec(json);
+            var spec = new PackageSpec();
             spec.Name = Name;
             spec.FilePath = FilePath;
-
+            spec.Title = Title;
+            spec.IsDefaultVersion = IsDefaultVersion;
+            spec.HasVersionSnapshot = HasVersionSnapshot;
+            spec.Description = Description;
+            spec.Summary = Summary;
+            spec.ReleaseNotes = ReleaseNotes;
+            spec.Authors = (string[]) Authors?.Clone();
+            spec.Owners = (string[]) Owners?.Clone();
```

Comment:
why do you need to clone these Authors or Owners?

Reply:
Why shouldn't they be cloned?